server/src/main/java/org/candlepin/hostedtest/HostedTestSubscriptionResource.java

```diff
@@ -188,6 +188,7 @@ public boolean deleteSubscription(@PathParam("subscription_id") String subscript
      * Deletes all subscriptions.
      */
     @DELETE
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Yeah, our listener throws a warning if the annotation is removed.  That's why it's there.